src/pudl/metadata/fields.py

```diff
@@ -2202,6 +2225,36 @@
         "description": "Production expenses: maintenance, supervision, and engineering (USD).",
         "unit": "USD",
     },
+    "opex_fgd_feed_materials_chemical": {
+        "type": "integer",
+        "unit": "dollars",
+        "description": "Annual operation and maintenance expenditures for feed materials and chemicals for FGD equipment, excluding electricity.",
+    },
```

Comment:
What are feed materials in this context?

Reply:
Also maybe change the last bit to excluding the cost of electricity?

Reply:
These are taken directly from the EIA form (same for below)